### ISSUE TITLE
Detected I/O inside the event loop

### ISSUE THREAD

**Issue by FaserF** [mammuth/ha-fritzbox-tools#158]
Hi,
I havent seen another issue with that Problem. I have updated to the newest version. Now Ha tells me the following:
```
Logger: homeassistant.util.async_
Source: util/async_.py:129
First occurred: 16:01:06 (4 occurrences)
Last logged: 16:02:07

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for fritzbox_tools doing I/O at custom_components/fritzbox_tools/binary_sensor.py, line 91: connection = self.fritzbox_tools.connection.call_action(
```

And after that:
```
Logger: custom_components.fritzbox_tools.binary_sensor
Source: custom_components/fritzbox_tools/binary_sensor.py:94
Integration: FRITZ!Box Tools (documentation)
First occurred: 16:01:06 (2 occurrences)
Last logged: 16:02:07

Error getting the state from the FRITZ!Box
Traceback (most recent call last):
  File "/config/custom_components/fritzbox_tools/binary_sensor.py", line 94, in _async_fetch_update
    is_up = await self.hass.async_add_executor_job(connection)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
TypeError: 'str' object is not callable
```

Had no issue for a while with this addon.

**Comment by chemelli74** [comment]
@FaserF I bet you are testing master branch, not official release.

Anyway, if you wait for #159 to be merged, problem is already fixed.

Simone

**Comment by TheZoker** [comment]
This should be fixed in the new release [1.15.1](https://github.com/mammuth/ha-fritzbox-tools/releases/tag/1.15.1), that is available now on hacs.

**Comment by FaserF** [comment]
Thank you for the quick fix. As I am only using HACS I would be surprised if I had used the master version
